machine/v1beta1/types_awsprovider.go

```diff
@@ -95,6 +95,17 @@ type AWSMachineProviderConfig struct {
 	// The field size should be greater than 0 and the field input must start with cr-***
 	// +optional
 	CapacityReservationID string `json:"capacityReservationId"`
+	// marketType specifies the type of market for the EC2 instance.
+	// Valid values are OnDemand, Spot, and CapacityBlock.
```

Comment:
I forgot that when on an optional field we also include "omitted" as an allowed value:
```suggestion
	// Valid values are OnDemand, Spot, CapacityBlock, and omitted.
```

Reply:
done